static/js/searchDocs.js

```diff
@@ -0,0 +1,39 @@
+function doClick() {
+       if(String(document.location.href).includes("/dataingest/upload")){
+         document.location.href = "../../datasheet-documentation";
```

Comment:
You could also probably do this with a dictionary object in way fewer lines, but that's not essential (although much cleaner)